pkg/server/router/presentation.go

```diff
@@ -0,0 +1,140 @@
+package router
+
+import (
+	"context"
+	"fmt"
+	"github.com/TBD54566975/ssi-sdk/credential/exchange"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"github.com/tbd54566975/ssi-service/pkg/server/framework"
+	svcframework "github.com/tbd54566975/ssi-service/pkg/service/framework"
+	"github.com/tbd54566975/ssi-service/pkg/service/presentation"
+	"net/http"
+)
+
+type PresentationDefinitionRouter struct {
+	service *presentation.Service
+}
+
+func NewPresentationDefinitionRouter(s svcframework.Service) (*PresentationDefinitionRouter, error) {
+	if s == nil {
+		return nil, errors.New("service cannot be nil")
+	}
+	service, ok := s.(*presentation.Service)
+	if !ok {
+		return nil, fmt.Errorf("could not create presentation router with service type: %s", s.Type())
+	}
+	return &PresentationDefinitionRouter{service: service}, nil
+}
+
+type CreatePresentationDefinitionRequest struct {
+	PresentationDefinition exchange.PresentationDefinition `json:"presentation_definition" validate:"required"`
+}
+
+type CreatePresentationDefinitionResponse struct {
+	PresentationDefinition exchange.PresentationDefinition `json:"presentation_definition"`
+}
+
+// CreatePresentationDefinition godoc
+// @Summary      Create PresentationDefinition
+// @Description  Create presentation definition
+// @Tags         PresentationDefinitionAPI
+// @Accept       json
+// @Produce      json
+// @Param        request  body      CreatePresentationDefinitionRequest  true  "request body"
+// @Success      201      {object}  CreatePresentationDefinitionResponse
+// @Failure      400      {string}  string  "Bad request"
+// @Failure      500      {string}  string  "Internal server error"
+// @Router       /v1/presentation/definition [put]
+func (sr PresentationDefinitionRouter) CreatePresentationDefinition(ctx context.Context, w http.ResponseWriter, r *http.Request) error {
+	var request CreatePresentationDefinitionRequest
+	if err := framework.Decode(r, &request); err != nil {
+		errMsg := "decoding"
```

Comment:
Fix the error message:
Invalid Presentation Definition Request 
or something like that